src/lib.rs

```diff
@@ -289,6 +305,43 @@ unsafe impl Slab for [MaybeUninit<u8>] {
     }
```

Comment:
A couple lines above I found the word `borrowck` - what does it mean?

Reply:
> Shorthand for borrowcheCKer

Yep